src/main/resources/application-cftlib.properties

```diff
@@ -7,33 +7,33 @@ spring.flyway.locations=classpath:db/migration,classpath:db/dev
 document_management.url=http://localhost:5005
 
 idam.api.url=http://localhost:5000
-idam.api.url.jwkUrl=http://localhost:5000/o/jwks
+idam.api.jwkUrl=http://localhost:5000/o/jwks
 idam.s2s-auth.url=http://localhost:8489
 idam.s2s-auth.totp_secret=AAAAAAAAAAAAAAAA
 idam.s2s-auth.microservice=et_cos
 
-ccd.data.store.api.url=http://localhost:4452
+ccd.data-store-api-url=http://localhost:4452
 
 # tornado-docker
 #tornado.url=http://localhost:9095/api/render
 # docmosis cloud service
 tornado.url=https://eu.dws3.docmosis.com/api/render
 
 case_document_am.url=http://localhost:4455
-rd_professional.api.url=http://localhost:4507
+rd_professional-api-url=http://localhost:4507
 feature.secure-doc-store.enabled=${SECURE_DOC_STORE_FEATURE:false}
 
 uk.gov.notify.api.key=${GOV_NOTIFY_API_KEY:someApiKey}
-referral.template.id=${REFERRAL_TEMPLATE_ID:12345}
-et1Serving.template.id=${ET1SERVING_TEMPLATE_ID:12345}
-et1Serving.respondent.template.id=${ET1SERVING_RESPONDENT_TEMPLATE_ID:12345}
-rejected.template.id=${REJECTED_TEMPLATE_ID:12345}
+template.referral=${REFERRAL_TEMPLATE_ID:12345}
```

Comment:
if all the base templates exist in application.yaml, maybe we don't need these version in cftlib?

Reply:
especially given that some are just dummy values

Reply:
and maybe cftlib gets the yaml treatment as well?

Reply:
Good spot, yes, can delete all these. I'll upgrade to yaml too